src/System.Drawing.Common/src/System/Drawing/GdiplusNative.Unix.cs

```diff
@@ -57,13 +68,18 @@ private static IntPtr LoadNativeLibrary()
                     foreach (var searchDirectory in searchDirectories)
                     {
                         var searchPath = Path.Combine(searchDirectory, libraryName);
-
+#if netcoreapp30
+                        if (NativeLibrary.TryLoad(searchPath, out lib))
+                        {
+                            break;
+                        }
+#else
                         lib = Interop.Libdl.dlopen(searchPath, Interop.Libdl.RTLD_LAZY);
-
                         if (lib != IntPtr.Zero)
                         {
                             break;
                         }
+#endif
                     }
                 }
 
```

Comment:
Switch dlsym below to use the managed API as well?

Reply:
Switched to use `NativeLibrary.GetSymbol` in `FunctionWrapper.cs`. In this file, `LoadFunctionPointer` method was unused.

Reply:
Switched to use `NativeLibrary.TryGetExport` in `FunctionWrapper.cs`. In this file, `LoadFunctionPointer` method was unused, so I have removed it.